WordPress/build.gradle

```
@@ -455,12 +465,41 @@ task violationCommentsToGitHub(type: se.bjurr.violations.comments.github.plugin.
     ]
 }
 
-task printVersionName {
+task printWordPressVersionName {
+    doLast {
+        println android.productFlavors.wordpress.versionName-rc-1
```

```suggestion
        println android.productFlavors.wordpress.versionName
```

I am guessing this was a copy/paste error.